testsuite/systemtest_long/procgen/ProGe/test_generate_debugger.testdesc

```diff
@@ -0,0 +1,10 @@
+<?php
```

Comment:
Please use the New TCE Systemtest Description Format, not the old php-based one. Run "find -name 'tcetest*.sh'" under systemtest for examples.